webui/react/src/components/MetricSelectFilter.tsx

```diff
@@ -45,7 +45,7 @@ const MetricSelectFilter: React.FC<Props> = ({
   onChange,
 }: Props) => {
   const [ filterString, setFilterString ] = useState('');
-  const selectRef = useRef<Select<SelectValue>>(null);
```

Comment:
not exactly sure what specific thing caused this (most likely the updated TypeScript version), but this type is no longer valid/accurate